docs/glossary-121.md

```diff
@@ -0,0 +1,106 @@
+---
+title: 121 Glossary
+hide:
+  - toc
+---
+
+A list of key terms used in the 121 platform.
+
+- [General terms](#general-terms)
+- [Tab Titles](#tab-titles)
+- [Program Cards](#program-cards)
+- [Registration Status](#registration-status)
+- [Payment Overview](#payment-overview)
+- [Payment Transfer statuses](#payment-transfer-statuses)
+
+### General terms
+
+| Term                   | Definition                           |
+| :------------------    | :------------------------------------------------------------------- |
+| **121 Platform**       | (“one to one”) is a platform that helps project managers by increasing efficiency in setting up and executing a cash program Information Management (IM) system.  |
+| **CVA**                | Cash and Voucher Assistance, referring to Cash Transfer Programming (CTP). |
+| **Project**            | A program created by users in the platform. |
+| **FSP**                | Financial Service Provider - an entity that provides financial services, which may include digital payment services.|
+| **Budget**             | Amount of money in the project that will be disbursed to included people/households.|
+| **Registrations**      | People or households that registered, subscribed to a project or programme.|
```

Comment:
I dont think we should mention programme here, and if we are: be consistent in program vs. programme